subscription_oca/views/sale_subscription_views.xml

```diff
@@ -370,12 +370,67 @@
         <field name="model">sale.subscription</field>
         <field name="arch" type="xml">
             <search>
+                <field
```

Comment:
**chore**: Could you reorder the tags for improved legibility please?

1. all the `<field>`s
2. all the `<filter>`s
3. all the groupbys (these are already ordered)